docs/imcce/imcce.rst

```diff
@@ -0,0 +1,496 @@
+.. doctest-skip-all
+
+.. _astroquery.imcce:
+
+******************************************************************************************************************************************
+Institut de Mécanique Céleste et de Calcul des Éphémérides (IMCCE) Solar System Services (`astroquery.imcce`/astroquery.solarsystem.imcce)
+******************************************************************************************************************************************
+
+Overview
+========
+
+IMCCE provides a number of Solar System-related services, two of which are currently implemented here:
+
+* `SkyBoT <http://vo.imcce.fr/webservices/skybot/>`_: search for and
+  identify Solar System objects that are present in a given area of
+  the sky at a given time
+* `Miriade <http://vo.imcce.fr/webservices/miriade/>`_: ephemerides service
+
+Use cases for both services are detailed below.
+  
+
+SkyBot - Solar System Body Identification Service
+=================================================
+
+Cone Search
+-----------
+
+`astroquery.imcce.SkybotClass` provides an interface to the `cone search
+<http://vo.imcce.fr/webservices/skybot/?conesearch>`_ offered by SkyBoT.
+
+A simple cone search for Solar System objects in a circular field
+looks like this:
+
+.. code-block:: python
+	  
+   >>> from astroquery.imcee import Skybot
+   >>> Skybot.cone_search((0, 0), 0.1, 2451200)  # doctest: +SKIP
+   <QTable length=6>
+   Number    Name            RA          ...      vy          vz       epoch  
+			    deg          ...    AU / d      AU / d       d    
+   int64     str9         float64        ...   float64     float64    float64 
+   ------ --------- -------------------- ... ----------- ----------- ---------
+   299383 2005 VC73   359.94077541666667 ... 0.010886508 0.003535381 2451200.0
+   518441  2004 FC6 0.048321666666666666 ... 0.006622583 0.004266802 2451200.0
+   395111 2009 VC31   359.97290749999996 ... 0.008183291 0.004832045 2451200.0
+   192116  2006 DM9         359.91487125 ...  0.00767636 0.005744082 2451200.0
+   239169 2006 KT47  0.09837624999999998 ... 0.009238599  0.00336863 2451200.0
+   359191 2009 CD65  0.09119958333333332 ... 0.009023276 0.001463182 2451200.0
+
+`~astroquery.imcce.SkybotClass.cone_search` produces a
+`~astropy.table.QTable` object with the properties of all Solar System
+bodies that are present in the cone at the given epoch. 
+   
+The required input arguments for
+`~astroquery.imcce.SkybotClass.cone_search` are the center coordinates of
+the cone, its radius, as well as the epoch of the
+observation. Coordinates ``coo`` can be provided as
+`~astropy.coordinates.SkyCoord` object, or as a tuple; in the latter
+case, the tuple has to consist of two floats that refer to RA and
+declination in degrees (all coordinates are given in J2000.0). The
```

Comment:
I'm hesitant to allow this sort of shorthand... I kind of like it because it's much easier than entering a SkyCoord, but this should be a "hidden expert-user feature" rather than the default; the documentation (above) should use the full and explicit SkyCoord approach.

Reply:
Agreed. `SkyCoords` will be used in all examples.